prow/config.yaml

```diff
@@ -1846,29 +1846,60 @@ periodics:
   name: ci-tekton-pipeline-upgrade-tests
   labels:
     preset-presubmit-sh: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    pipeline-kind-e2e: "true"
   agent: kubernetes
   extra_refs:
   - org: tektoncd
     repo: pipeline
     base_ref: main
     path_alias: github.com/tektoncd/pipeline
   spec:
+    affinity:
+      podAntiAffinity:
```

Comment:
It seems like this is saying it cannot run on the same node as pipeline e2e tests, why is this? can you add some comments here? Same goes for the combinations of nodeSelector and tolerations.

Reply:
Thanks to @dibyom , this is added at https://github.com/tektoncd/plumbing/issues/1281